src/import-export/bin/export-data

```diff
@@ -23,16 +23,19 @@ print_usage()
 {
 	echo "Usage:"
 	echo "    $COMMAND [OPTIONS]"
-	echo "    Export data suitable for migrating servers. By default this"
-	echo "    includes the Nextcloud database, configuration, and data"
-	echo "    (equivalent to running $COMMAND -abcd)."
+	echo "    Export Nextcloud data suitable for migrating servers."
+	echo "    By default this includes the"
+        echo "    apps, database, config, certs, keys, and data"
+	echo "    (equivalent to running $COMMAND -abcdef)."
 	echo ""
 	echo "Available options:"
 	echo "    -h: Display this help message"
 	echo "    -a: Include the (non-default) apps"
 	echo "    -b: Include the database"
 	echo "    -c: Include the config"
 	echo "    -d: Include the data (can be quite large)"
+	echo "    -e: Include the encryption keys"
+	echo "    -f: Include the certificates"
```

Comment:
Can we please split the export of keys and certs into two PRs (and two issues)?

Reply:
Oh yes, that would make more sense wouldn't it.  I'm not very advanced with github but I'll give it a go!  This PR will probably end up with some extra commits to delete the certificates bit while I put it in a different PR.  I have no idea how to clean that up.

Reply:
Done.  Let me know if I did that right.